rhc-ose-ansible/roles/openshift_generate_inventory/templates/inventory_template.j2

```diff
@@ -0,0 +1,67 @@
+# Create an OSEv3 group that contains the master, nodes, etcd, and lb groups.
+# The lb group lets Ansible configure HAProxy as the load balancing solution.
+# Comment lb out if your load balancer is pre-configured.
+[OSEv3:children]
+masters
+nodes
+etcd
+lb
```

Comment:
This one should probably have the same conditional as below - i.e. "{% if openshift_lbs is defined %}"